hack/util.sh

```diff
@@ -345,4 +357,26 @@ function util::get_load_balancer_ip() {
   return 1
 }
 
+# util::add_routes will add routes for given kind cluster
+# Parameters:
+#  - $1: name of the kind cluster want to add routes
+#  - $2: the kubeconfig path of the cluster wanted to be connected
+#  - $3: the context in kubeconfig of the cluster wanted to be connected
+function util::add_routes() {
+  echo "start connecting cluster"
+  unset IFS
+  echo "get route rules"
+  kubectl --kubeconfig ${2} --context ${3} get nodes -o jsonpath='{range .items[*]}ip route add {.spec.podCIDR} via {.status.addresses[?(.type=="InternalIP")].address}{"\n"}'
+  routes=$(kubectl --kubeconfig ${2} --context ${3} get nodes -o jsonpath='{range .items[*]}ip route add {.spec.podCIDR} via {.status.addresses[?(.type=="InternalIP")].address}{"\n"}')
```

Comment:
I do not quite understand why to execute the same command twice `kubect ...`

Reply:
sorry. I add it because I want to get more detail infos before. And I forget to delete.